src/backend/language.cpp

```diff
@@ -65,249 +86,130 @@ LanguageMap::LanguageMap() {
     */
     QObject::tr("Names of languages", "No need to translate - see the longer comment (If there is no longer comment, it doesn't work yet :)) ------ ");
 
-    auto const addLanguage =
-        [this](QStringList abbrevs, QString englishName) {
-            auto language =
-                    std::make_shared<Language>(std::move(abbrevs),
-                                               std::move(englishName));
-            for (auto const & abbrev : language->abbrevs()) {
-                BT_ASSERT(!contains(abbrev));
-                insert(abbrev, language);
```

Comment:
This seems to have been converted from a lambda to a `LanguageMap` member function by accident due to an incorrect merge request?